module/builder/consensus/config.go

```diff
@@ -3,28 +3,21 @@
 package consensus
 
 import (
-	"time"
+	"github.com/onflow/flow-go/consensus/hotstuff"
 )
 
 type Config struct {
-	minInterval time.Duration
-	maxInterval time.Duration
+	blockTimestamp hotstuff.BlockTimestamp
 	// the max number of seals to be included in a block proposal
 	maxSealCount      uint
 	maxGuaranteeCount uint
 	maxReceiptCount   uint
 	expiry            uint
 }
 
-func WithMinInterval(minInterval time.Duration) func(*Config) {
+func WithBlockTimestamp(blockTimestamp hotstuff.BlockTimestamp) func(*Config) {
```

Comment:
same renaming suggestion: `BlockTimestamp` -> `Blocktimer`
```suggestion
func WithBlocktimer(timer hotstuff.Blocktimer) func(*Config) {
```